yarn-project/archiver/src/archiver/archiver.test.ts

```diff
@@ -402,7 +407,7 @@ describe('Archiver', () => {
     expect(loggerSpy).toHaveBeenCalledWith(`No blocks to retrieve from 1 to 50`);
 
     // Lets take a look to see if we can find re-org stuff!
-    await sleep(1000);
+    await sleep(2000);
```

Comment:
Hm, why the sleep bump?

Reply:
It was needed for it to pass. Guess the KvArchiveStore is slower.